### PR TITLE
Video 9513 update disabled video UI

### PR DESCRIPTION
**Pull request by olipyskoty**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VIDEO-9513](https://issues.corp.twilio.com/browse/VIDEO-9513)

### Description

This PR blurs a participant's video whenever their video is switched off. After two seconds, a short message is displayed over the blurred video letting the user know that the video has been switched off to conserve bandwidth. This is so there isn't a quick flash of the message if the video is only switched off for a brief moment. 

Also, I've updated the storybook controls so that you can unpublish/publish and switch on/switch off the video tracks individually, or a specific list of fake participants. This is the same behavior as the screenshare storybook controls.

Finally, comparing this branch against the grid view feature branch since `main` does not have storybook.

### Before review
* [ ] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [x] Included screenshot as PR comment (if needed)
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

### CODE REVIEW

File: src/components/MainParticipantInfo/MainParticipantInfo.tsx [hunk 1]
```diff
@@ -104,8 +104,26 @@ const useStyles = makeStyles((theme: Theme) => ({
...
+  },
+  '@keyframes showMessage': {
+    '0%': {
+      opacity: 0,
+      visibility: 'hidden',
+    },
+    '100%': {
+      opacity: 1,
+      visibility: 'visible',
+    },
```
Comment (timmydoza) [inline src/components/MainParticipantInfo/MainParticipantInfo.tsx:123]: With something like this, we can give the message a nice fade in:
```
'@keyframes showMessage': {
      '0%': {
        opacity: 0,
        visibility: 'hidden',
      },
      '80%': {
        opacity: 1,
        visibility: 'visible',
      },
      '100%': {
        opacity: 1
      },
    }
```

No need to use these exact values. Play with it and see what works.

File: src/components/ParticipantInfo/ParticipantInfo.tsx [hunk 2]
```diff
@@ -121,6 +134,28 @@ const useStyles = makeStyles((theme: Theme) =>
         fontSize: '0.75rem',
       },
     },
+    switchedOffMessage: {
```
Comment (timmydoza) [inline src/components/ParticipantInfo/ParticipantInfo.tsx:137]: This text could also have a shadow to make it more readable. Without it, you wouldn't be able to read white text on a white background. I don't know much about text-shadow, but something like this seemed ok: `textShadow: '0 0 3px rgba(0, 0, 0, 0.7)',`

More info here: https://css-tricks.com/almanac/properties/t/text-shadow/

**Review by timmydoza — CHANGES_REQUESTED** [review]

File: src/components/MainParticipantInfo/MainParticipantInfo.tsx (hunk 1 above)
Comment (olipyskoty) [reply src/components/MainParticipantInfo/MainParticipantInfo.tsx:123]: I like the idea of fading the message in, but I'm not sure I'm seeing much of a difference in the UI using your example. I tried changing `opacity` to something like `0.8` for `80%` and that looked kind of glitchy. I did notice that increasing the `animationDuration` from `0.5s` to `1s` created a fade in effect. I'm wondering if you're seeing something that I'm not seeing though, so you might need to show me?

Comment (timmydoza) [reply src/components/MainParticipantInfo/MainParticipantInfo.tsx:123]: Hmm, I wasn't originally seeing a fade in, but now I am. Not sure what changed, but it looks good now :+1:

File: src/components/ParticipantInfo/ParticipantInfo.tsx [hunk 3]
```diff
@@ -136,8 +136,9 @@ const useStyles = makeStyles((theme: Theme) =>
     },
     switchedOffMessage: {
       opacity: 0,
+      textShadow: '0 0 3px rgba(0, 0, 0, 0.7)',
```
Comment (timmydoza) [inline src/components/ParticipantInfo/ParticipantInfo.tsx:139]: Same, let's add this to MainParticpantInfo too.

**Review by timmydoza — CHANGES_REQUESTED** [review]

**Comment by olipyskoty** [comment]
Created new PR in #731 to avoid large merge conflict

**Pull request closed without merging**